### PR TITLE
feat: Make settings page responsive

### PR DESCRIPTION
**Pull request by BrunoQuaresma**
Screen Shot 2022-07-26 at 16 29 08
Screen Shot 2022-07-26 at 16 29 00

Related to #3185

### CODE REVIEW

File: site/src/components/TabSidebar/TabSidebar.tsx [hunk 1]
```diff
@@ -47,10 +47,22 @@ const useStyles = makeStyles((theme) => ({
   menu: {
     minWidth: 160,
     marginTop: theme.spacing(5),
+
+    [theme.breakpoints.down("sm")]: {
+      display: "flex",
+      borderBottom: `1px solid ${theme.palette.divider}`,
+      marginBottom: theme.spacing(2),
+      // 48px is the side margins
+      width: "calc(100% + 48px)",
```
Comment (presleyp) [inline site/src/components/TabSidebar/TabSidebar.tsx:56]: can we use a variable for the 48px?

**Review by presleyp — APPROVED** [review]
Looks really nice!

File: site/src/components/TabSidebar/TabSidebar.tsx (hunk 1 above)
Comment (BrunoQuaresma) [reply site/src/components/TabSidebar/TabSidebar.tsx]: Totally!

**Comment by AbhineetJain** [comment]
I believe we can add a story for the reduced viewport if it does not exist.

**Pull request merged**